package.json

```diff
@@ -11,6 +11,7 @@
     "build-storybook": "storybook build"
   },
   "dependencies": {
+    "clsx": "^2.1.1",
```

Comment:
If you will use specific libraries `^2.1.1` -> `2.1.1`  it will make you code more stable